spec/dummy/.controlplane/controlplane.yml

```diff
@@ -0,0 +1,200 @@
+aliases:
+  common: &common
+    cpln_org: shakacode-heroku-to-control-plane-ci
```

Comment:
Add comment that a developer can override this by specifying ENV value of CPLN_ORG.

Reply:
Need to document that the test can only run if the developer has full access to the org.

Thus, update CONTRIBUTING.md on how to develop and run tests.